### PR TITLE
Database creation scripts

### PR DESCRIPTION
**Pull request by ArtyomKolosov2**
Added three level architecture projects and DbCreator class for creating database using T-SQL.

### CODE REVIEW

File: WebAutopark/WebAutopark.BusinessLogic/Program.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace WebAutopark.BusinessLogic
+{
+    class Program
+    {
+        static void Main(string[] args)
```
Comment (PushMeNow) [inline WebAutopark/WebAutopark.BusinessLogic/Program.cs:7]: I don't realize why do you need console app here??

File: WebAutopark/WebAutopark.DataAccess/Program.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace WebAutopark.DataAccess
+{
+    class Program
```
Comment (PushMeNow) [inline WebAutopark/WebAutopark.DataAccess/Program.cs:5]: Again tell me the reason of this decision (console app)?

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi.
I didn't realize why have you created console app? Please fix it or tell me the reason.

File: WebAutopark/WebAutopark.DataAccess/Program.cs (hunk 2 above)
Comment (ArtyomKolosov2) [reply WebAutopark/WebAutopark.DataAccess/Program.cs:5]: Yeah, my bad. But I've already fixed it

**Review by PushMeNow — APPROVED** [review]
Approved

**Pull request merged**